src/pages/[platform]/build-a-backend/add-aws-services/analytics/existing-resources/index.mdx

```diff
@@ -97,4 +98,3 @@ Amazon Pinpoint requires an AWS Identity and Access Management (IAM) policy in o
 }
 ```
 
-</InlineFilter>
```

Comment:
Can you please explain why this moved?

Reply:
the `## Update your IAM Policy` section is applicable to all platforms. the JS platform section should start after the `## Update your IAM Policy`